.github/actions/environment/action.yml

```diff
@@ -8,6 +8,11 @@ runs:
         echo TEST_TIMEOUT=310 | tee -a $GITHUB_ENV
         echo NO_OUTPUT_TIMEOUT=320 | tee -a $GITHUB_ENV
         echo PRESERVE_ENVVARS=REPLICATION_SYNC_TIMEOUT,TEST_TIMEOUT,NO_OUTPUT_TIMEOUT | tee -a $GITHUB_ENV
+        # For self-hosted runners run w/o restart may need to kill
+        # hanged processes that could be leaved from the previous
+        # workflows runs.
+        ps auxT | grep " tarantool " | awk '{system("kill -9 "$2)}' || true
```

Comment:
Will processes from a previous run attached to the same terminal? Will tarantool processes with redirected stdio attached to a terminal at all?

Reply:
As I know the terminal will be the other, but ps will see all the needed processes to remove.

Reply:
This does not look as a proper explanation, but since you removed it, okay.